docs/examples/general/expressions/expr_examples.ipynb

```diff
@@ -4,26 +4,32 @@
    "cell_type": "markdown",
```

Comment:
<p>Same, point 2 is not aligned</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I don't see it in the source. Maybe some rendering issue? I'll make sure that it is ok in the final build when I have it.</p>